### PR TITLE
Mico development

### PR DESCRIPTION
**Pull request by tmicomyiza**
I updated the README file to describe the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+This repo contains Mico's portfolio and STEP projects.
+This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program
```
Comment (danhoshow) [inline README.md:4]: nit: add a period at the end

File: README.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+
+This file has been updated
```
Comment (danhoshow) [inline README.md:6]: Was this line required?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
 # Google Student Training in Engineering Program
 
 This repo contains Mico's portfolio and STEP projects.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program
```
Comment (danhoshow) [inline README.md:4]: nit: add a period at the end

File: README.md (hunk 2 above)
Comment (tmicomyiza) [reply README.md:6]: 3rd line was required, but 6 wasn't

**Pull request closed without merging**